### PR TITLE
Update README.md - add Koyeb's managed data service

### PR DESCRIPTION
**Pull request by alisdairbr**
## Requirements

 * [X] This is Software as a Service not self hosted
 * [X] It has a free tier not just a free trial
 * [X] Pricing information is clearly visible without signup or phone calls
 * [X] The submission mentions what is free
 * [X] The submission is not already present in the list
 * [X] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1251,6 +1251,7 @@ Update Time, five active automations, webhooks.
    * [FaunaDB](https://fauna.com/) — Serverless cloud database with native GraphQL, multi-model access, and daily free tiers up to 100 MB
    * [HarperDb](https://harperdb.io/) — Serverless cloud database, with dynamic schema based on JSON, 3000 IOPS with 1GB storage
    * [Upstash](https://upstash.com/) — Serverless Redis with free tier up to 10,000 requests per day, 256MB max database size, and 20 concurrent connections
+   * [Koyeb](https://www.koyeb.com/) - Free Managed Postgres database available in Frankfurt (Germany), Washington, D.C. (US), and Singapore. 1GB memory, 1GB storage, and 0.25 CPU.
```
Comment (ripienaar) [inline README.md:1254]: We list services only once, so you'll have to add it to the most appropriate single place.

Comment (alisdairbr) [reply README.md:1254]: Gotcha, thanks! Will go ahead and mention this in the PaaS section.

Comment (alisdairbr) [reply README.md]: just updated this :)

**Comment by ripienaar** [comment]
thanks

**Pull request merged**